### PR TITLE
feat(main): There where no way to configure log level

### PR DESCRIPTION
**Pull request by ybelMekk**
* We used zap.NewProductionConfig (default zap.InfoLevel), but always override with zap.DebugLevel

* added to config

### CODE REVIEW

File: cmd/digdirator/main.go [hunk 1]
```diff
@@ -41,7 +42,12 @@ func init() {
 	_ = nais_io_v1.AddToScheme(scheme)
 	// +kubebuilder:scaffold:scheme
 
-	log.SetLevel(log.DebugLevel)
+	lvl, err := log.ParseLevel(viper.GetString(config.LogLevel))
+	if err != nil {
+		log.SetLevel(log.InfoLevel)
+	} else {
+		log.SetLevel(lvl)
+	}
```
Comment (tronghn) [inline cmd/digdirator/main.go:50]: Looks like the if and else here are flipped

**Review by tronghn — APPROVED** [review]
lgtm otherwise 👍

File: cmd/digdirator/main.go (hunk 1 above)
Comment (tronghn) [reply cmd/digdirator/main.go]: Wait, nevermind - it was correct. Too early for me 😄

Comment (ybelMekk) [reply cmd/digdirator/main.go]: haha :D

Comment (ybelMekk) [reply cmd/digdirator/main.go]: Skal lage en annen bedre if

**Pull request merged**